### PR TITLE
Feature/selection page improvement

### LINKED ISSUE #21: Build a Statistics / Progress Page
As a User,
I want to keep track of my progress,
so I need a statistics page.

AC:

- [x] Build a Progress page
- [x] connect the progress data per lesson

### LINKED ISSUE #27: Improve the Vocabulary Set Selection Page
As a User, 
I want a nicer page to look at which also provides more information,
so I need a better selection page

AC:

- [x] Build a custom Checkbox component to select the chapters
- [x] Add a stats visualisation to the component
- [x] Build a nicer Layout

### PR DESCRIPTION
**Pull request by sebuzz**
## Motivation

As a User,
I want to keep track of my progress,
so I need a statistics page.

AC:

 Build a Progress page
 connect the progress data per lesson

__

As a User,
I want a nicer page to look at which also provides more information,
so I need a better selection page

AC:

 Build a custom Checkbox component to select the chapters
 Add a stats visualisation to the component
 Build a nicer Layout

## Issues closed

closes #21 
closes #27

### CODE REVIEW

File: src/organisms/chapters-card/index.jsx [hunk 1]
```diff
@@ -1,65 +1,86 @@
...
+		let knowledgeLevel = (sum / thisLessonShown.length) * 24 + 24;
+		console.log("KnLevel:", knowledgeLevel, "lesson:", number);
+		// calculate the percentage of viewed cards of the current lesson
+		const percent = thisLessonShown.length / numberOfWords;
+		console.log("percent:", percent);
+		// return both
+		return { knowledgeLevel, percent };
+	};
+	console.log("calculate...", calculateBoxColor(7));
+
```
Comment (konrascher3) [inline src/organisms/chapters-card/index.jsx:45]: console.logs

File: src/organisms/chapters-card/index.jsx [hunk 2]
```diff
@@ -1,65 +1,86 @@
...
+								{/*		color: "black",*/}
+								{/*		margin: 0,*/}
+								{/*		alignContent: "center",*/}
+								{/*	}}*/}
+								{/*	onClick={() => {*/}
+								{/*		setMyChapter(item.id);*/}
+								{/*	}}*/}
+								{/*>*/}
+								{/*	{item.id}*/}
+								{/*</button>*/}
```
Comment (konrascher3) [inline src/organisms/chapters-card/index.jsx:75]: unused-code

File: src/organisms/chapters-card/index.jsx [hunk 3]
```diff
@@ -1,65 +1,86 @@
...
+							</ChapterSelector>
+						);
+					})}
+				</ChapterGrid>
+
+				{/*<CardFooter />*/}
```
Comment (konrascher3) [inline src/organisms/chapters-card/index.jsx:81]: Hier auch

**Review by konrascher3 — APPROVED** [review]

File: src/organisms/chapters-card/index.jsx (hunk 1 above)
Comment (sebuzz) [reply src/organisms/chapters-card/index.jsx:45]: OK, will delete

File: src/organisms/chapters-card/index.jsx (hunk 2 above)
Comment (sebuzz) [reply src/organisms/chapters-card/index.jsx:75]: OK

File: src/organisms/chapters-card/index.jsx (hunk 3 above)
Comment (sebuzz) [reply src/organisms/chapters-card/index.jsx:81]: okay

**Pull request merged**